Kajabi/sage-lib
# Nav and NavLink - updated rails docs

## Description

- [x] - update demo to use rails code
- [x] - a11y added prop to include `aria-label`

## Test notes

No visual changes

### Steps for testing
1. Visit the sage rails nav page: http://localhost:4000/pages/object/nav
2. Verify the updates to docs and demo
3. Visit the sage rails nav link page: http://localhost:4000/pages/element/nav_link
